app/config/config.neon

```diff
@@ -50,11 +44,12 @@ php:
 	date.timezone: Europe/Prague
 
 tracy:
-    email: xxx@xxx.xx
+#	email: xxx@xxx.xx
 
 services:
 	#cache.storage: Nette\Caching\Storages\FileStorage(%tempDir%)
 	cache.storage: Nette\Caching\Storages\SQLiteStorage('%tempDir%/cache.db')
 
+	api.core.errorHandler: Apitte\Core\ErrorHandler\PsrLogErrorHandler(Tracy\Bridges\Psr\TracyToPsrLoggerAdapter())
```

Comment:
Tohle je klicove.

Reply:
Díky zdá sa že to funguje. Dúfam, že nie ako pred tým keď to logovalo len niekedy. Ale dúfam, že nie. Ešte raz vďaka.